src/components/ContactGroups/ContactGroupCreation.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react'
+import { Icon, Input, translate } from 'cozy-ui/react'
+import classNames from 'classnames'
+import './index.styl'
+
+const normalizeGroupData = name => {
+  return {
+    name: name,
+    metadata: {
+      version: 1
+    }
+  }
+}
+class ContactGroupCreation extends Component {
+  state = {
+    isInput: false,
+    groupName: '',
+    focused: false
+  }
+  constructor(props) {
+    super(props)
+    this.textInput = React.createRef()
+  }
+  handleClick = () => {
+    this.setState({ isInput: !this.state.isInput })
+  }
+  onFocus = e => {
+    e.stopPropagation()
+  }
+
+  onClick = e => {
+    e.stopPropagation()
+  }
+  keyPress = async e => {
+    if (e.keyCode == 13) {
+      e.stopPropagation()
+      this.props.createGroup(normalizeGroupData(e.target.value))
+      this.textInput.current.value = ''
+    }
+    if (e.keyCode === 8) {
+      e.stopPropagation()
+    }
+    //space key is used to validate an option
+    if (e.keyCode === 32) {
+      e.stopPropagation()
+    }
+    e.stopPropagation()
```

Comment:
I think we can remove the previous calls to `stopPropagation` since this one is always called?